internal/server/web/proxy/vllm.go

```diff
@@ -213,7 +212,6 @@ func logVllmCompletionRequest(log *zap.Logger, cr *vllm.CompletionRequest, prod,
 
 	if prod {
 		fields := []zapcore.Field{
-			zap.String(logFiledNameCorrelationId, cid),
```

Comment:
cid can br removed from the function header here and the `logVllmChatCompletionRequest` function